crates/bevy_app/src/app.rs

```diff
@@ -440,8 +440,9 @@ impl App {
 
     /// Sets the main runner loop function for this Bevy App
     ///
-    /// Usually the main loop is handled by Bevy integrated plugins ([`WinitPlugin`]), but
-    /// in some cases one might wish to implement their own main loop.
+    /// Usually the main loop is handled by Bevy integrated plugins 
+    /// ([`WinitPlugin`](https://docs.rs/bevy_winit/0.5.0/bevy_winit/struct.WinitPlugin.html)), 
```

Comment:
full links to docs.rs should be avoided

Reply:
What should I change it to get the correct link? Replace the 0.5.0 with *?
We can't point it to WinitPlugin from inside that crate, because it doesn't have that as a dependency.

Reply:
you should be able to add a dev dependencies to `bevy_winit` that will allow doc links to work 😃

Reply:
I don't think it does... It doesn't bring it into scope for the doc comments. I'm fairly certain we'd have to bring it in as a full dependency.
https://github.com/rust-lang/api-guidelines/discussions/186

Oh here we are! I was trying to dig and find this! https://discord.com/channels/691052431525675048/743559241461399582/871090893162156082

Reply:
Cargo doesn't provide dev-dependencies when documenting. @nemo157 has suggested `doc-dependencies` a few times but it doesn't currently exist.

See also https://github.com/rust-lang/rust/issues/74481.

Reply:
Oh right, dev dependencies are available when testing doc code blocks but not when building docs...

The other place where we have a similar link is https://github.com/bevyengine/bevy/blob/90586a4c461cde86b8a0e08f0770894e7cf30862/crates/bevy_ecs/src/event.rs#L120

If we consider than the main target for those links will be docs.rs users, I think it should link to the root crate (`bevy` instead of `bevy_winit`) and use `*` for version. What do you think?

Reply:
I think that sounds good, I'll go through and do that.